packages/bulkapi/src/controllers/BulkApi.ts

```diff
@@ -56,6 +76,7 @@ export class BulkApiImpl implements BulkApi {
     }
 
     deletePartitions = async (req: Request, res: Response): Promise<void> => {
+        logger.requestLog(` * deletePartitions request received, with body of ${req.headers["content-length"]} bytes`)
         const idList = req.body
```

Comment:
Could it make sense to return 404, if the requested iud does not exist?

Reply:
Yes, I am currently only returning 400 in case of an error, did not look into any other 40* code. In this case there isn't even a 400 status, so I added the 404.